contracts/orchestrator-contracts/tsconfig.json

```diff
@@ -1,10 +1,13 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.options.json",
```

Comment:
Shouldn't this change be in `tsconfig.build.json` instead?

Reply:
I followed [the guide](https://moonrepo.dev/docs/guides/javascript/typescript-project-refs#on-all-projects), and everything is working as expected